File: C++ With DSA/Chapters/Day3/grade.py
"""
marks >= 90 -> A
marks are 90-80 -> B
makrs are less then 80 -> C

"""
marks = int(input("Enter marks: "))
if(marks >= 90):
    print("A")
elif(marks >= 80 and marks < 90):
    print("B")
else:
    print("C")

File: C++ With DSA/Chapters/Day3/grade.cpp
#include <iostream>
using namespace std;

/*
marks are up to 90 -> A
marks are up to 80 and down to 90 -> B
marks are up to 33 and down to 80 -> C
marks are down to 33 -> F
*/

int main() {
    int marks;
    cout<<"Enter marks: ";
    cin>>marks;
    if(marks >= 90){
        cout<<"Your grade is A";
    } else if(marks >= 80 && marks < 90){
        cout<<"Your grade is B";
    } else if(marks >= 33 && marks < 80){
        cout<<"Your grade is C";
    } else{
        cout<<"F!";
    }
    return 0;
}